### PR TITLE
Implements errors sorting

### LINKED ISSUE #12: Sort exceptions in the Sidebar
At the moment, exceptions in the sidebar are sorted by time (latest exceptions first). Sometimes it is useful to be able to sort by other criteria.

Implement a dropdown on the top of the sidebar that allows selecting the sorting type (by latest occurrence, by count). Implement the sorting logic that obeys this option.

### PR DESCRIPTION
**Pull request by sikozonpc**
Fixes #12 

*Implements:
 - Sorting dropdown button for the errors on the sidebar.

Dev notes from this PR:
 - created a `utils/errors` with some utilities for errors manipulation.
 - for the dropdown sorting used the bootstrap one with default styling.
 - added the sorting button in a 'actions' div where maybe will be more buttons such as Searching.
 - if the errors sidebar has many items until it can scroll the actions div should be fixed on top (maybe for a future pr).

### CODE REVIEW

File: web/src/components/SideBar.tsx [hunk 1]
```diff
@@ -18,10 +19,17 @@ interface ConnectedProps {
 
 interface DefaultsProps {
   errors: AggregatedError[],
+  activeSortFilter: SortFilters,
+  setActiveErrorSortFilter: (filter: SortFilters) => void,
```
Comment (jcreixell) [inline web/src/components/SideBar.tsx]: I believe this is not needed here, as it is already defined in `DispatchProps` and mapped by `matchDispatchToProps`.

note that `handleErrorSelect: (errorKey: string) => void` is a left over as we have not moved it to the reducer yet, so it needs to be passed in `App.tsx` when creating the component. But in this case we don't need to do that.

**Review by jcreixell — APPROVED** [review]
@sikozonpc thank you very much for your contribution! 🥇 

The change looks great, I added a small comment, mostly a nitpick.

Regarding fixing the actions to the top, it could be a nice future improvement indeed 🔝

File: web/src/components/SideBar.tsx (hunk 1 above)
Comment (sikozonpc) [reply web/src/components/SideBar.tsx]: Indeed, I'll also move `activeSortFilter: SortFilters` to the `ConnectedProps`

**Comment by marctc** [comment]
Thanks for this @sikozonpc!!! Don't forget to add yourself in the list of AUTHORS

**Pull request merged**